test/TorchSharpTest/TestTorchTensor.cs

```diff
@@ -16,6 +16,12 @@ namespace TorchSharp
 {
     public class TestTensor
     {
+        public TestTensor()
+        {
+            // Avoid comma issues on international machines (Swedish, for instance...)
+            CultureInfo.DefaultThreadCurrentCulture = (CultureInfo)CultureInfo.InvariantCulture.Clone();
+        }
+
```

Comment:
This should never be needed. You should fix test code instead.

Reply:
The test code, for expediency, expects US numbers formatting in all the ToString() tests.

Reply:
The culture needs to be passed to `ToString` then.

Reply:
It's because of tests like below, they  expect there to be a "." as a decimal separator, but I have a "," as a decimal separator.

Did you mean I should flip the culture in every one of the tests that fail (seven) and flip it back after? Or did you have some other idea?

                Assert.Equal($"[2x2x2x4], type = Float32, device = cpu{_sep}{_sep}[0,0,:,:] ={_sep}        0   3.141 6.2834 3.1415{_sep}" +
                             $" 6.28e-06 -13.142   0.01 4713.1{_sep}{_sep}[0,1,:,:] ={_sep} 0.01 0 0 0{_sep}    0 0 0 0{_sep}{_sep}" +
                             $"[1,0,:,:] ={_sep}        0   3.141 6.2834 3.1415{_sep} 6.28e-06 -13.142   0.01 4713.1{_sep}{_sep}" +
                             $"[1,1,:,:] ={_sep} 0.01 0 0 0{_sep}    0 0 0 0{_sep}", str);

Reply:
I missed the conversation above, I'll fix this by passing in to string